### ISSUE TITLE
cannot resolve ascii-graphs dependency

### ISSUE THREAD

**Issue by bastiion** [JetBrains/sbt-idea-plugin#18]
I tried to develop a new sbt-based intellij idea plugin and I cannot find any repository containing org.jetbrains ascii-graphs.

**Comment by mutcianm** [comment]
Hi! Seems some artifacts haven't been propagated to the sbt repository yet.
You can try manually adding original repository to `plugins.sbt`
```Scala
resolvers += Resolver.url("jetbrains-bintray",
  url("https://dl.bintray.com/jetbrains/sbt-plugins/"))(Resolver.ivyStylePatterns)
```

However, this plugin is undergoing an overhaul at the moment and there are some critical issues(#17 
 #14 #13), especially if you plan to depend on Scala plugin for IDEA.
I would estimate it'll take one more week to get all things resolved.

**Comment by mutcianm** [comment]
Please check again, the package is up

**Comment by regadas** [comment]
Addin the `resolver` solves the issue 👍

**Comment by mutcianm** [comment]
`ascii-graphs` dependency is published to maven since plugin version 3.0.0
